lighthouse-cli/bin.ts

```diff
@@ -241,8 +242,15 @@ function showProtocolTimeoutError() {
   process.exit(_PROTOCOL_TIMEOUT_EXIT_CODE);
 }
 
+function showPageLoadError() {
+  console.error('Unable to load the page.');
```

Comment:
maybe add something friendly here about rechecking the URL and that the page is actually working?